homeassistant/components/harmony/strings.json

```diff
@@ -19,7 +19,7 @@
       "unknown": "[%key:common::config_flow::error::unknown%]"
     },
     "abort": {
-      "already_configured": "[%key:common::config_flow::abort::already_configured_device%]"
+      "already_configured_device": "[%key:common::config_flow::abort::already_configured_device%]"
```

Comment:
This should be reverted. The key needs to be `already_configured` to map to our helper method that aborts the flow if an existing entry has the same unique id.